### PR TITLE
fix: extra pixels section overflow

### PR DESCRIPTION
**Pull request by Oshioke-Salaki**
- [ ] issue #184
- [ ] follows contribution [guide](https://github.com/keep-starknet-strange/art-peace/blob/main/CONTRIBUTING.md)
- [ ] code change includes tests
- [ ] breaking change

Fixed overflow issue in extra pixels tab

### CODE REVIEW

**Review by b-j-roberts — CHANGES_REQUESTED** [review]
This doesn't seem to solve the issue, please test using docker compose.

Make sure to use the latest changes in main, and use
`docker compose down --volumes`, `docker compose build`, and `docker compose up`.
Then wait for the app to completely start, not just the frontend.

Then you can work within the frontend directory and changes will auto update in the UI. Sometimes a refresh on the tab is needed

**Comment by Oshioke-Salaki** [comment]
> This doesn't seem to solve the issue, please test using docker compose.
> 
> Make sure to use the latest changes in main, and use `docker compose down --volumes`, `docker compose build`, and `docker compose up`. Then wait for the app to completely start, not just the frontend.
> 
> Then you can work within the frontend directory and changes will auto update in the UI. Sometimes a refresh on the tab is needed

Okay sir 
I will submit a PR today
Sorry i for the delay. I haven't been feeling too well

File: frontend/src/tabs/canvas/ExtraPixelsPanel.css [hunk 1]
```diff
@@ -58,6 +58,8 @@
   justify-content: center;
 
   /* TODO: Fix scrolling */
+  height: 100%;
+  max-height: 240px;
```
Comment (b-j-roberts) [inline frontend/src/tabs/canvas/ExtraPixelsPanel.css:62]: Were you able to test this?

Comment (Oshioke-Salaki) [reply frontend/src/tabs/canvas/ExtraPixelsPanel.css:62]: No I wasn't. Does it work now? But it should work

Comment (b-j-roberts) [reply frontend/src/tabs/canvas/ExtraPixelsPanel.css:62]: No it doesn't

Comment (b-j-roberts) [reply frontend/src/tabs/canvas/ExtraPixelsPanel.css:62]: You can hard code the users extra pixels and faction pixels data to something in app.js if that helps testing too.

**Review by b-j-roberts — CHANGES_REQUESTED** [review]
Nice, looks like the scrolling is working well. Couple of things :

Could you set it so the "Available" header is outside the scrolling div and stays at the top?

Also, there are now overflow issues with the factions info. Could you fix this?
image

**Pull request closed without merging**